SimulationCore/Deutsch/Insekt.cs

```diff
@@ -20,79 +20,79 @@ public override int Id
         /// </summary>
         public int AktuelleEnergie
         {
-            get { return ((Simulation.CoreInsect)element).AktuelleEnergieBase; }
+            get { return ((Simulation.CoreInsect)element).currentEnergyBase; }
         }
 
         /// <summary>
         /// Gibt die aktuelle geschwindigkeit an
         /// </summary>
         public int AktuelleGeschwindigkeit
         {
-            get { return ((Simulation.CoreInsect)element).AktuelleGeschwindigkeitBase; }
+            get { return ((Simulation.CoreInsect)element).CurrentSpeedBase; }
         }
 
         /// <summary>
         /// Gibt die Angriffsstärke an
         /// </summary>
-        public int Angriff
+        public int Angriffstaerke
```

Comment:
This one will break the API of the Ant. Here we need to stick with "Angriff"

Reply:
Danke für die Rückmeldung.
ich bin auch kein Native Speaker aber ich bin beim Compilieren mit Rider auf einige Probleme mit den Umlauten gestoßen, daher der Versuch die Simulation-Classes auf Englisch gerade zu ziehen.

Eine Frage auf die ich beim übersetzten gestoßen bin: 
ist Range für Movement Range oder für Smell Range oder für beides?
CasteInfo.cs Zeile 102
            // Prüfen, ob der Riechweitemodifikator im Rahmen ist
            if (Range < SimulationSettings.Custom.CasteSettings.MinIndex ||
                Range > SimulationSettings.Custom.CasteSettings.MaxIndex)
in allen anderen Zusammenhängen, die ich bisher gesehen habe war Range die Bewegungsreichweite und nicht die Riechreichweite
Update obiger Files folgt

Reply:
Bei dem ganzen deutschen Quellcode und den Kommentaren brennen mir inzwischen ja auch die Augen :D Deshalb bin ich so froh, dass du dich dieser Sache annimmst.

Was die Range angeht: Es gibt im Grunde ja keine Smell Range. Die Ameise riecht eine Markierung, sobald deren Radius den Kollisionsradius der Ameise kreuzt. Wie weit die Ameise also riechen kann, ist nur von der Markierung abhängig. Daher handelt es sich hier um die maximale Laufreichweite der Ameise.